backend-api/template.yaml

```diff
@@ -327,7 +335,16 @@ Resources:
                   - kms:Decrypt
                   - kms:GenerateDataKey
                 Resource: !GetAtt TxMAKMSEncryptionKey.Arn
-      RoleName: !Sub ${AWS::StackName}-token-lambda
+        - PolicyName: ENIPolicy
```

Comment:
Does this name makes sense for the modified lambda roles in this PR? @sandy-may

Reply:
Maybe VpcConfigurationPolicy / VpcConfigPolicy / VpcAccessPolicy?